lib/emscripten/src/emscripten_target.rs

```diff
@@ -173,3 +173,393 @@ pub fn ___gxx_personality_v0(ctx: &mut Ctx, a: i32, b: i32, c: i32, d: i32, e: i
     debug!("emscripten::___gxx_personality_v0");
     0
 }
+// round 2
+pub fn nullFunc_dii(ctx: &mut Ctx, index: i32) {
+    debug!("emscripten::nullFunc_dii");
+}
+pub fn nullFunc_diiii(ctx: &mut Ctx, index: i32) {
+    debug!("emscripten::nullFunc_diiii");
+}
+pub fn nullFunc_iiji(ctx: &mut Ctx, index: i32) {
+    debug!("emscripten::nullFunc_iiji");
+}
+pub fn nullFunc_j(ctx: &mut Ctx, index: i32) {
+    debug!("emscripten::nullFunc_j");
+}
+pub fn nullFunc_jij(ctx: &mut Ctx, index: i32) {
+    debug!("emscripten::nullFunc_jij");
+}
+pub fn nullFunc_jjj(ctx: &mut Ctx, index: i32) {
+    debug!("emscripten::nullFunc_jjj");
+}
+pub fn nullFunc_vd(ctx: &mut Ctx, index: i32) {
+    debug!("emscripten::nullFunc_vd");
+}
+pub fn nullFunc_viiiiiii(ctx: &mut Ctx, index: i32) {
+    debug!("emscripten::nullFunc_viiiiiii");
+}
+pub fn nullFunc_viiiiiiii(ctx: &mut Ctx, index: i32) {
+    debug!("emscripten::nullFunc_viiiiiiii");
+}
+pub fn nullFunc_viiiiiiiii(ctx: &mut Ctx, index: i32) {
+    debug!("emscripten::nullFunc_viiiiiiiii");
+}
+pub fn nullFunc_viiij(ctx: &mut Ctx, index: i32) {
+    debug!("emscripten::nullFunc_viiij");
+}
+pub fn nullFunc_viiijiiii(ctx: &mut Ctx, index: i32) {
+    debug!("emscripten::nullFunc_viiijiiii");
+}
+pub fn nullFunc_viiijiiiiii(ctx: &mut Ctx, index: i32) {
+    debug!("emscripten::nullFunc_viiijiiiiii");
+}
+pub fn nullFunc_viij(ctx: &mut Ctx, index: i32) {
+    debug!("emscripten::nullFunc_viij");
+}
+pub fn nullFunc_viiji(ctx: &mut Ctx, index: i32) {
+    debug!("emscripten::nullFunc_viiji");
+}
+pub fn nullFunc_viijiii(ctx: &mut Ctx, index: i32) {
+    debug!("emscripten::nullFunc_viijiii");
+}
+pub fn nullFunc_viijj(ctx: &mut Ctx, index: i32) {
+    debug!("emscripten::nullFunc_viijj");
+}
+pub fn nullFunc_vij(ctx: &mut Ctx, index: i32) {
+    debug!("emscripten::nullFunc_vij");
+}
+pub fn nullFunc_viji(ctx: &mut Ctx, index: i32) {
+    debug!("emscripten::nullFunc_viji");
+}
+pub fn nullFunc_vijiii(ctx: &mut Ctx, index: i32) {
+    debug!("emscripten::nullFunc_vijiii");
+}
+pub fn nullFunc_vijj(ctx: &mut Ctx, index: i32) {
+    debug!("emscripten::nullFunc_vijj");
+}
+pub fn invoke_dii(ctx: &mut Ctx, index: i32, a1: i32, a2: i32) -> f64 {
```

Comment:
The `invoke_*` functions are intended to be used as a `try`, `catch` analog. We don't support that kind of thing right now, but we could think about registering another trap handler here? We should look into registering multiple trap handlers and checking each one.